sgf-viewer/src/content.ts

```diff
@@ -0,0 +1,97 @@
+        "content": [
+            {
+                "title": "KataGo without search (level of top 100 European player)",
+                "dir_name": "no_search",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/2782FF00582288C8.sgfs",
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/2797913CD0AAD6C1.sgfs",
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/27A3CFF58C2E44BE.sgfs",
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/282923DD52BC6029.sgfs",
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/298DA53FEAF14D4B.sgfs",
+                    "/nas/ucb/ttseng/go_attack/match/adv-checkpoints/33-extra/sgfs/2A02D8CCF5291DAB.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "34.1 million training steps, 600 visits",
+                "victim": "Latest (cp505-v1-MCTS), no search",
+                "description": [
+                    "Even without search, Katago's <code>Latest</code> network plays at the <a href=\"https://arxiv.org/pdf/2211.00241.pdf#page=16\">strength of a top-100 European professional</a>. Yet our adversary achieves a 99% win rate against this victim by playing a counterintuitive strategy. The adversary stakes out a minority territory in the corner, allowing KataGo to stake the complement, and placing weak stones in KataGo’s stake.",
+                    "KataGo predicts a high win probability for itself and, in a way, it’s right—it would be simple to capture most of the adversary’s stones in KataGo’s stake, achieving a decisive victory. However, KataGo plays a pass move before it has finished securing its territory, allowing the adversary to pass in turn and end the game. This results in a win for the adversary under the standard <a href=\"https://tromp.github.io/go.html\">Tromp-Taylor</a> ruleset for computer Go, as the adversary gets points for its corner territory (devoid of victim stones) whereas the victim does not receive points for its unsecured territory because of the presence of the adversary’s stones."
+                ]
+            },
+            {
+                "title": "KataGo with 64 visits (level of top world professionals)",
+                "dir_name": "64_visits",
+                "server": "dqn.ist.berkeley.edu",
+                "_path_comment": "Computed by `grep -l v=1024 * | head -n 10` in /nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs, see https://www.notion.so/chaiberkeley/match-cp505-64-vs-adv-1_to_8192-39e0e303cd3f45199f4ce638c92556a5 for experiment log",
```

Comment:
should be computed by `grep v=8192`, not `grep v=1024`, right?

Reply:
Yes, seems right, see above.